### DISCUSSION TITLE
RustDesk - open source alternative to TeamViewer

### DISCUSSION THREAD

**Discussion by tOOmki** [tteck/Proxmox#3691] [Ideas]
Please add:
https://rustdesk.com/docs/en/self-host/

**Comment by pos-ei-don** [comment]
+1

**Comment by unixversal** [comment]
+1

**Comment by miketrump** [comment]
+1
Very good alternative to Teamviewer and Anydesk

**Comment by level42ca** [comment]
+1

Please, this would be very much appreciated!!! <3

**Comment by Beo666** [comment]
+1

**Comment by kiwinh0** [comment]
+1

**Comment by TagorePde** [comment]
why not use MeshCentral? is much better.

**Reply by pos-ei-don** [reply]
Not the Android App....

**Reply by tOOmki** [reply]
https://rustdesk.com/docs/en/client/android/

**Comment by covers007** [comment]
Yes! It's very good idea. Integration please 😉

**Comment by xkpx64** [comment]
+1 for **Alpine** LXC (already running fine with me)
